index.ts

```diff
@@ -0,0 +1,31 @@
+import * as modelCollection from './lib/model-collection';
+import * as defaultDynamicModels from './lib/dynamic';
+import * as defaultObservationModels from './lib/observation';
+
+function camelToDash(str: string) {
```

Comment:
Original export are named using Dash syntax, but this syntax cause export issues, so intread of exporting `my-dynamic-model` I export `myDynamicModel` the convert it back to the Dash syntax to keep the existing syntax.